### DISCUSSION TITLE
Not able to bypass /admin/users/me/permissions API on v3.1.4

### DISCUSSION THREAD

**Discussion by vinod77** [strapi/strapi#7546] [Legacy Topics]
**Describe the bug**
Not able to bypass` /admin/users/me/permissions` API

We want to bypass it since we already using our own Authentication using SSO. We are not use Strapi's default authentication

**Steps to reproduce the behavior**

1. On Strapi v3.1.4
2. Use your own Authentication using SSO (not Strapi's default Authentication)
3. Try to run the app 
3. `/admin/users/me/permissions` is throwing 403 forbidden error

**Expected behavior**
Should be able to bypass `/admin/users/me/permissions` API, so we can use our own Authentication using SSO

**System**

- Node version: v10.15.3
- NPM version: 6.4.1
- Strapi version: 3.1.4
- Database: Mongo DB `v4.0.3`
- Operating system: MAC

**Comment by derrickmehaffy** [comment]
Hello,

I see you are wanting to ask a question that is not really a bug report or a feature request, questions should be directed to the [discussions](https://github.com/strapi/strapi/discussions). Please see the following contributing guidelines for asking a question [here](https://github.com/strapi/strapi/blob/master/CONTRIBUTING.md#reporting-an-issue).

I have converted this to a discussion for you, we try to keep Github clean and only for bug reports and feature requests.

**Comment by derrickmehaffy** [comment]
Hello, 

We will need more information as we can't really help if we don't understand the implementation of your custom SSO, do you have a Github, Gitlab, Bitbucket repo that you can link?

**Comment by vinod77** [comment]
@derrickmehaffy Thanks for the response.
Actually I cannot share my code as it's a private/corporate.
In older Strapi versions (until 3.0) we bypassed Strapi default Authentication and used our own SSO Authentication on Strapi `/admin`.  But on Strapi `v3.1.4` I am want to Bypass this Strapi default Authentication with my own custom Authentication.
Could you please provide me steps to override Strapi Authentication? I see it's getting called from `strapi-admin` backed and getting used in `strapi-helper-plugin`.
Thank you!

**Reply by derrickmehaffy** [reply]
Currently we cannot recommend a way to do this right now without you forking the `strapi-admin` package. We do plan to support SSO for the Admin: https://portal.productboard.com/strapi/1-roadmap/c/61-sso-authentication-providers but the ETA is not til the end of the year. We do also plan to make the admin package a plugin (to allow for backend extensions) but there is no ETA on that: https://github.com/strapi/strapi/issues/4838
